tools/wake/main.cpp

```diff
@@ -608,6 +623,11 @@ int main(int argc, char **argv) {
     }
   }
 
+  if (alerted_slow_cache) {
```

Comment:
This isn't correct, you need the `i + 1` above instead

Reply:
the `i+1` isn't why this is needed.

The print above only occurs every 1 second, which means that we will almost never print the last 100 or so without this

Reply:
Ah I see gotcha.

Another idea that we could do if you want: We could employ `\b` here to make this look really fancy btw. Then The message would just disappear instead of needing to be printed out.

Reply:
Personal preference is to leave the message instead of clearing it